aws-lc-fips-sys/builder/main.rs

```diff
@@ -124,6 +143,10 @@ fn get_cmake_config(manifest_dir: &PathBuf) -> cmake::Config {
 fn prepare_cmake_build(manifest_dir: &PathBuf, build_prefix: Option<&str>) -> cmake::Config {
     let mut cmake_cfg = get_cmake_config(manifest_dir);
 
+    if OutputLibType::default() == OutputLibType::Dynamic {
+        cmake_cfg.define("BUILD_SHARED_LIBS", "1");
```

Comment:
Probably should force this to `0` in the else branch. Not sure if we want to rely on the cmake default behavior here to be correct.